libbeat/cmd/keystore.go

```diff
@@ -186,6 +186,9 @@ func addKey(store keystore.Keystore, keys []string, force, stdin bool) error {
 
 	key := strings.TrimSpace(keys[0])
 	value, err := store.Retrieve(key)
+	if err != nil {
```

Comment:
I'm not sure we want to return here.  One of the errors could be "ErrKeyDoesntExists", which is fine.  The function is trying to add a new key, so skipping that error is what we want to do.  Is there a bug this is addressing?